test/topology_experimental_raft/test_tablets.py

```diff
@@ -646,3 +646,117 @@ async def check():
 
     tablet_count = await get_tablet_count(manager, servers[0], 'test', 'test')
     assert tablet_count > 1
+
+async def assert_tablet_count_metric_value_for_shards(manager: ManagerClient, server: ServerInfo, expected_count_per_shard: list[int]):
+    tablet_count_metric_name = "scylla_column_family_tablet_count"
```

Comment:
It seems, that the name of metric will need to be adjusted when Amnon's PR is merged. Perhaps, PR#17182 has higher priority.

Ref:
https://github.com/scylladb/scylladb/pull/17182

I will rebase the change and adjust this line when the mentioned PR is merged.

Reply:
The branch was rebased and the name of tested metric has been adjusted .